src/backend/web/routes/post.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const { getPost } = require('../../utils/storage');
+
+const post = express.Router();
+
+post.get('/:id', async (req, res) => {
+  const redisPost = await getPost(req.params.id);
+
+  /**
+   * If the post retrieved from redis doesn't have the property 'author'
```

Comment:
I don't fully understand this comment.  It seems odd to use `author` as a sentinel like this.  Can we change how our storage code works to return null instead?